src/main/java/org/gridsuite/modification/server/entities/ModificationGroupEntity.java

```diff
@@ -39,8 +39,14 @@ public ModificationGroupEntity(UUID uuid) {
     }
 
     public void addModification(ModificationEntity modification) {
-        modifications.add(modification);
-        modification.setGroup(this);
+        if (modifications.isEmpty()) {
```

Comment:
Maybe add a comment to explain we have to do that to maintain proper order with @OrderColumn annotation

Reply:
done